packages/recurring-payments-job/src/__mocks__/dynamics-web-api.js

```diff
@@ -0,0 +1,3 @@
+import { configureDynamicsWebApiMock } from '../../../dynamics-lib/src/__mocks__/dynamics-web-api-mock-helper.js'
+const DynamicsWebApi = jest.genMockFromModule('dynamics-web-api')
+export default configureDynamicsWebApiMock(DynamicsWebApi)
```

Comment:
I'm not a fan of auto mocks - they hide functionality and make things work 'magically' in an opaque way. Mocks should be declared in the same file as the tests they're serving.